api/src/main/java/org/datacleaner/configuration/DataCleanerEnvironment.java

```diff
@@ -31,6 +31,13 @@
 public interface DataCleanerEnvironment {
 
     /**
+     * Gets the {@link RemoteServerConfiguration}.
+     *
+     * @return the box for credentials defined for this environment.
+     */
+    public RemoteServerConfiguration getRemoteServerConfiguration();
```

Comment:
Can the server configuration be used for anything else than the descriptor provider? If not, then maybe it should not be exposed/contained within the ```DataCleanerEnvironment``` interface (and corresponding class) but only be an implementation detail of the ```RemoteDescriptorProviderImpl```.

As far as I can see, the server configuration isn't used for anything else?